tetris.py

```diff
@@ -39,6 +40,16 @@
     # this file is.
     image_dir = pathlib.Path(__file__).parent / "images"
 
+try:
+    with open(image_dir / "game_data.json", "r") as source:
+        json_dict = json.load(source)
+except FileNotFoundError:
+    # TODO: game_data.json shouldn't be in image_dir
```

Comment:
Fix this instead of copying bad code around.

Reply:
Actually, ThePhilgrim fixed this already on minesweeper, but he's not responding to my PR review on minesweeper. The correct code is in his PR.

Reply:
Deleted try part for now